runtime/compiler/control/CompilationThread.cpp

```diff
@@ -94,6 +93,7 @@
 #include "env/SystemSegmentProvider.hpp"
 #include "env/DebugSegmentProvider.hpp"
 #ifdef COMPRESS_AOT_DATA
+#include "shcdatatypes.h" // For CompiledMethodWrapper
```

Comment:
I think you need a `#endif` here right?

Reply:
The `#endif` already exists, together with the `#ifdef` and a large section of code in between. I just migrated the include from outside the guard to inside the guard because it's only needed when `COMPRESS_AOT_DATA` is defined.

Reply:
Ooh right